src/main/java/frc/robot/Constants.java

```diff
@@ -58,10 +58,10 @@ public static class ShooterConstants {
     public static final double SHOOTER_LEFT_D = 0;
     public static final double SHOOTER_LEFT_FF = 0;
 
-    public static final double SHOOTER_P = 0.001;
-    public static final double SHOOTER_I = 0.00000035;
-    public static final double SHOOTER_D = 0;
-    public static final double SHOOTER_FF = 0;
+    public static final double SHOOTER_RIGHT_P = 0.001;
```

Comment:
You will want to merge from main.  There is a new HowdyPID object that is defined for shooter's pid

Reply:
Resolved in bb39de9